### ISSUE TITLE
[Hack-together] (feat): allow users to see house rating from home page

### ISSUE THREAD

**Issue by VictorCodebase** [Azure-Samples/contoso-real-estate#322]
**Are you accessing the project from the GitHub Codespaces?**

- [ ✅] Yes, I am using GitHub Codespaces.

> Note: The project is optimized to run on Codespaces. We welcome contributions to enhance it for cross-platform local development, but it's not something we will prioritize right now. Thank you!

I really hope its not too late to suggest a feature.
I noticed its need when working on the other, thats why its really late

## A little background
I noticed the need for this while working on the comment section. I needed to look for a well reviewed (lie 5 stars) place (like 5 stars) to see if the comment section was working as intended. 
I however ended up having to click listing after listing - waiting for screens to load just to get something as simple as its rating
I then wondered how that may affect a user who wants a good house but has to go wait on multiple load screens.

**This may not be very hard to implement; I should be done by the end of today. Hopefully😂🥲**

## Feature description
A Lising's rating is displayed nest to its name to allow users to make pick choices faster.

## Screenshots
![image](https://github.com/Azure-Samples/contoso-real-estate/assets/135356007/0a0a44b8-f7a9-4d1e-b442-9c2db4984815)

![Screenshot 2023-08-30 174638](https://github.com/Azure-Samples/contoso-real-estate/assets/135356007/90a9737f-aa8b-42ae-8b2a-035608a06687)

To view on Figma:
https://www.figma.com/file/54sMZ0kHDvlandOk0unGvd/star-rating-on-home-page?type=design&node-id=1%3A2&mode=design&t=TITJ9syTibf0TcwD-1

## response
Im more than open to suggestions on how to improve scss or any other aspect. Thanks!!

**Comment by VictorCodebase** [comment]
My PR is almost there. Should I submit it once done😃?

https://github.com/Azure-Samples/contoso-real-estate/assets/135356007/5341539f-2677-4c02-a6fd-d31dd68d1abf

**Comment by VictorCodebase** [comment]
My PR is officially ready; Im soo excited😅. I've not done unnecessary formatting. 

Making it was a lot tougher than I imagined but it was for sure fun.

Should I create a PR😃?

screenshots:

![Screenshot 2023-08-31 082524](https://github.com/Azure-Samples/contoso-real-estate/assets/135356007/4bc6c6da-448f-486d-9838-94b56decada0)


![image](https://github.com/Azure-Samples/contoso-real-estate/assets/135356007/f9807772-71b6-4bca-807a-b2812beb06ec)

**Comment by anfibiacreativa** [comment]
Closed. Out of scope.
